mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -256,6 +256,15 @@ class LogisticRegression @Since("1.2.0") (
   @Since("1.5.0")
   override def getThresholds: Array[Double] = super.getThresholds
 
+  /**
+   * Set suggested depth for treeAggregate or treeReduce (>= 2).
+   * Default is 2.
+   * @group setParam
```

Comment:
As DB Tsai said in the JIRA, it needs another task to take longer to discuss about the expert param. Shall we do it in this PR?